### ISSUE TITLE
Val Accuracy

### ISSUE THREAD

**Issue by dongfengxijian** [sail-sg/VGT#6]
Thank you for sharing your programe. When I trained the model without CM-pretain under default setting, I got val accuracy of 54.22. How can I reproduce 55.02 which is presented in the paper? Thank you for your help!

![微信截图_20230425204009](https://user-images.githubusercontent.com/24863806/234278708-1cf062f2-6e54-4b62-a2e4-55c784245141.png)
![微信截图_20230425204407](https://user-images.githubusercontent.com/24863806/234279740-7888493a-c7a9-4627-bfd4-febac8b9ab7e.png)

**Comment by doc-doc** [comment]
Not sure what your problem is, have you tried to freeze the BERT parameters and fine-tune the best model obtained at the 1-st stage?

**Comment by dongfengxijian** [comment]
Thank you for your replying. I just ran ./shells/next_train.sh 0 with the default setting and got the result. When I trained the model on two gpus, it got val accuracy of  54.48%. Should I fine-tune the best model which I got by running  ./shells/next_train.sh 0?

**Comment by doc-doc** [comment]
Yes. You also need to uncomment the parameter-freeze part as suggested in model/language_model.py

**Comment by dongfengxijian** [comment]
Thank you for your patient guidance！
